sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationAudience.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Data.AppConfiguration
+{
+    /// <summary> Cloud audiences available for AppConfiguration. </summary>
+    public readonly partial struct ConfigurationAudience : IEquatable<ConfigurationAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConfigurationAudience"/> object.
+        /// </summary>
+        /// <param name="value">The Azure Active Directory audience to use when forming authorization scopes.
```

Comment:
```suggestion
        /// <param name="value">The Entra ID audience to use when forming authorization scopes.
```

I think?  We should get Scott Addie to confirm branding.

Reply:
Use the wording "Microsoft Entra audience".